docs/Utils.md

```diff
@@ -18,4 +19,18 @@ Return list of currencies exchange rates.
 | --- | --- | --- |
 | currencies | `Array<Object{from, to}>` | Array of objects where `from` property represents currency you have and `to` - currency you want. |
 
-**See: <a href="../examples/Utils/currencyConvert">Currency convert example</a>**
+**See: <a href="../examples/Utils/currencyConvert.js">Currency convert example</a>**
+
+<a name="refdata"></a>
+## .referenceDataReq(params)
+Returns list of Reference Data 
+
+**Returns**: `Promise`
+**See**: [Reference Data](https://support.travelport.com/webhelp/uapi/Content/Getting_Started/Design_Considerations/Reference_Data.htm)
+
+| Param | Type | Description |
+| --- | --- | --- |
+| DataType | `String` | Types of Reference Data as specified from the support page. <a href="./ReferenceDataTypes.md">See List</a> |
```

Comment:
It's better to keep same naming convention for param names being camelCase instead of PascalCase

Reply:
Absolutely.. current commit is done with the changes